cli/src/commands/obslog.rs

```diff
@@ -28,7 +28,7 @@ use crate::formatter::Formatter;
 use crate::graphlog::{get_graphlog, Edge};
 use crate::ui::Ui;
 
-/// Show how a change has evolved
+/// Show revisions of the same Change-Id
```

Comment:
This isn't quite accurate. We have talked about walking the operation log to find how commits have been rewritten (something like `jj op log --change-id=xyz`, perhaps), but that's not how `jj obslog` works. `jj obslog` instead uses [a separate `predecessors` field that's stored in the commit](https://github.com/martinvonz/jj/blob/c8123e1555b450483cd5964ad7999bc9c9e4126a/lib/src/backend.rs#L131) (but not in the actual git commit when using the git backend). It can point to a predecessor with a different change id. `jj squash` actually points to both/all input commits.

Maybe "Show how a revision has been rewritten" or "Show past incarnations of revision" or something like that?

Reply:
"Show how a revision has been evolved over time"?

Reply:
That also work for me, thanks

Reply:
Updated!